CONTRIBUTING.md

```diff
@@ -146,3 +146,30 @@ and triggers release and publishing based on the following rules:
 | Commit with type `feat`            | Minor release |
 | Commit with type `fix`             | Patch release |
 | Commit with type `perf`            | Patch release |
+
+## Link widgets with local version of components/sdk-component-adapter
+
+When contributing to one of the dependent repositories like components or the
+sdk-component-adapter, it may be helpful to be able to test the changes done
+to the dependent repo in a "real-life" scenario like the meeting widget. In
+order to do that, you would have to check out both widgets and the dependent
+repo, and list the dependent package as a local dependency
+(see https://docs.npmjs.com/cli/v7/configuring-npm/package-json#local-paths).
+
+Below you can find an example for linking a local `components` package to the widgets.
+
+First, in your local `components` folder, you need to run `npm run build` to generate the "dist"
```

Comment:
```suggestion
1. Clone `components` repository
    ``bash
     git clone git@github.com:webex/components.git
    ````
2. Build `components` code. This will generate a `dist` folder
    ```bash
    cd components
    npm install
    npm run build
    ```
```